src/hermite_multidimensional.hpp

```diff
@@ -22,6 +27,14 @@
 typedef unsigned long long int ullint;
 
 
+/**
+ * TODO
+ *
+ * @param pos
+ * @param resolution
+ *
+ * @return
+ */
```

Comment:
@bgupt, could you fill in this function docstring, thanks!

Reply:
Done!